### ISSUE TITLE
[Show] Method with [Inline]

### ISSUE THREAD

**Issue by ajboni** [vexe/VFW#17]
Hello Again!
If I try to use [Inline] with a BetterBehaviour which has a [Show] attribute on a method I get the following error:

```
NullReferenceException: Object reference not set to an instance of an object
Vexe.Editor.Drawers.RecursiveDrawer.DrawRecursive (System.Object& target, Vexe.Editor.GUIs.BaseGUI gui, Int32 id, UnityEngine.Object unityTarget, System.String[] memberNames) (at Assets/Plugins/Editor/Vexe/Drawers/API/Core/RecursiveDrawer.cs:292)
Vexe.Editor.Drawers.RecursiveDrawer.DrawRecursive (System.Object target, Vexe.Editor.GUIs.BaseGUI gui, Int32 id, UnityEngine.Object unityTarget, System.String[] memberNames) (at Assets/Plugins/Editor/Vexe/Drawers/API/Core/RecursiveDrawer.cs:250)
Vexe.Editor.Drawers.InlineDrawer.DrawTarget (UnityEngine.Object target) (at Assets/Plugins/Editor/Vexe/Drawers/User/Others/InlineDrawer.cs:212)
Vexe.Editor.Drawers.InlineDrawer.OnLowerGUI () (at Assets/Plugins/Editor/Vexe/Drawers/User/Others/InlineDrawer.cs:201)
Vexe.Editor.GUIs.BaseGUI.Member (Vexe.Editor.Types.EditorMember member, System.Attribute[] attributes, Vexe.Editor.Drawers.BaseDrawer memberDrawer, Boolean ignoreComposition) (at Assets/Plugins/Editor/Vexe/GUIs/BaseGUI/Controls/Members.cs:152)
Vexe.Editor.GUIs.BaseGUI.Member (Vexe.Editor.Types.EditorMember member, System.Attribute[] attributes, Boolean ignoreComposition) (at Assets/Plugins/Editor/Vexe/GUIs/BaseGUI/Controls/Members.cs:60)
Vexe.Editor.GUIs.BaseGUI.Member (Vexe.Editor.Types.EditorMember member, Boolean ignoreComposition) (at Assets/Plugins/Editor/Vexe/GUIs/BaseGUI/Controls/Members.cs:53)
Vexe.Editor.GUIs.BaseGUI.Member (System.Reflection.MemberInfo member, System.Object rawTarget, UnityEngine.Object unityTarget, Int32 id, Boolean ignoreComposition, Vexe.Editor.Types.EditorMember& wrappedMember) (at Assets/Plugins/Editor/Vexe/GUIs/BaseGUI/Controls/Members.cs:42)
Vexe.Editor.GUIs.BaseGUI.Member (System.Reflection.MemberInfo info, System.Object rawTarget, UnityEngine.Object unityTarget, Int32 id, Boolean ignoreComposition) (at Assets/Plugins/Editor/Vexe/GUIs/BaseGUI/Controls/Members.cs:21)
Vexe.Editor.Internal.MembersCategory.Draw (UnityEngine.Object target) (at Assets/Plugins/Editor/Vexe/Editors/Internal/MembersCategory.cs:121)
Vexe.Editor.Editors.BaseEditor.OnGUI () (at Assets/Plugins/Editor/Vexe/Editors/BaseEditor.cs:409)
Vexe.Editor.GUIs.RabbitGUI.OnGUI (System.Action guiCode, Vector2 padding, Int32 targetId) (at Assets/Plugins/Editor/Vexe/GUIs/RabbitGUI/RabbitGUI.cs:191)
Vexe.Editor.Editors.BaseEditor.OnInspectorGUI () (at Assets/Plugins/Editor/Vexe/Editors/BaseEditor.cs:155)
UnityEditor.InspectorWindow.DrawEditor (UnityEditor.Editor editor, Int32 editorIndex, Boolean forceDirty, System.Boolean& showImportedObjectBarNext, UnityEngine.Rect& importedObjectBarRect, Boolean eyeDropperDirty) (at C:/buildslave/unity/build/Editor/Mono/Inspector/InspectorWindow.cs:1150)
UnityEditor.DockArea:OnGUI()
```

Test Class:

```
`using Vexe.Runtime.Types;
using UnityEngine;
using System.Collections;

public class TEST :  BetterBehaviour {
    [Show] 
    public void TestMethod() {
        Debug.Log("Test");
    }

}
```

Instance

```
    [Inline]
    public TEST test;
```

Thanks!!

**Comment by vexe** [comment]
Fixed in 1.3.2 which I will release soon

If you want to apply the fix, go to that line and add:

<code>
if (em != null) <br>
    target = em.RawTarget;
</code>

That portion of the code is very hacky and not pretty, but it was a must to get struct editing to work

**Comment by ajboni** [comment]
Worked like a charm! Thanks again!!!
